example/ck_tile/16_batched_gemm/run_batched_gemm_example.inc

```diff
@@ -188,15 +188,62 @@ int run_batched_gemm_example_with_layouts(int argc,
         c_m_n_gpu_ref.SetZero();
         c_m_n_gpu_buf_ref.SetZero();
 
+        //// TEST
+        ADataType* d_A;
+        BDataType* d_B;
+        CDataType* d_C;
+
+        hipError_t errA = hipMalloc(&d_A, batch_count * M * K * sizeof(ADataType));
+        hipError_t errB = hipMalloc(&d_B, batch_count * N * K * sizeof(BDataType));
+        hipError_t errC = hipMalloc(&d_C, batch_count * M * N * sizeof(CDataType));
+        if(errA != hipSuccess)
```

Comment:
Can we add some macro for these checks? (I suppose that it exist somewhere)

Reply:
You're right, we have a function for that, fixed in latest change